src/enclave/enclave.h

```diff
@@ -117,17 +114,6 @@ namespace enclave
       ::memcpy(node_cert, r.first.node_cert.data(), r.first.node_cert.size());
       *node_cert_len = r.first.node_cert.size();
 
-      if (r.first.quote.size() > quote_size)
-      {
-        LOG_FAIL_FMT(
-          "Insufficient space ({}) to copy quote out ({})",
-          quote_size,
-          r.first.quote.size());
-        return false;
-      }
-      ::memcpy(quote, r.first.quote.data(), r.first.quote.size());
```

Comment:
I'm betting this `memcpy` is why we previously had `quote = {1}`.

Reply:
Yes it was.